src/js/jsx/sections/style/Type.jsx

```diff
@@ -597,31 +641,31 @@ define(function (require, exports, module) {
                         <div className="control-group column-10 control-group__vertical">
                             <SplitButtonList size="column-10" className="button-radio__fixed">
                                 <SplitButtonItem
-                                    disabled={this.props.disabled}
+                                    disabled={locked}
                                     iconId="text-left"
                                     className={"split-button__item__fixed"}
                                     selected={alignment === "left"}
                                     onClick={this._handleAlignmentChange.bind(this, textLayer.alignmentTypes.LEFT)}
                                     title={strings.TOOLTIPS.ALIGN_TYPE_LEFT} />
                                 <SplitButtonItem
-                                    disabled={this.props.disabled}
+                                    disabled={locked}
                                     iconId="text-center"
                                     className={"split-button__item__fixed"}
                                     selected={alignment === "center"}
                                     onClick={this._handleAlignmentChange.bind(this, textLayer.alignmentTypes.CENTER)}
                                     title={strings.TOOLTIPS.ALIGN_TYPE_CENTER} />
                                 <SplitButtonItem
-                                    disabled={this.props.disabled}
+                                    disabled={locked}
                                     iconId="text-right"
                                     className={"split-button__item__fixed"}
                                     selected={alignment === "right"}
                                     onClick={this._handleAlignmentChange.bind(this, textLayer.alignmentTypes.RIGHT)}
                                     title={strings.TOOLTIPS.ALIGN_TYPE_RIGHT} />
                                 <SplitButtonItem
+                                    disabled={locked || !box}
```

Comment:
Gonna push back on this one: I think that grouping `box` with the other collection operations makes more sense than moving it someplace closer